nullaway/src/main/java/com/uber/nullaway/fixserialization/qual/AnnotationConfig.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022 Uber Technologies, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.uber.nullaway.fixserialization.qual;
+
+/** Container object of any {@code @Nonnull} and {@code @Nullable} annotations. */
+public class AnnotationConfig {
+
+  private Annotation nonNull;
+  private Annotation nullable;
+
+  /** Container object of annotation. */
+  public static class Annotation {
+    /** Simple name. */
+    public String name;
+    /** Fully qualified name. */
+    public String fullName;
+
+    private Annotation(String fullName) {
+      this.fullName = fullName;
+      String[] paths = fullName.split("\\.");
+      this.name = paths[paths.length - 1];
+    }
+
+    /**
+     * returns string representation of content of an object.
+     *
+     * @return string representation of contents of an object in a line seperated by tabs.
+     */
+    public String tabSeparatedToString() {
```

Comment:
Why this vs overridding `toString()`?

Reply:
c0c5c81fb28859957d3ca6d48fbd005e50ff01ef

Wanted to have this consistency that all elements involved in serialization work with this method, but in this small case I do agree that we can just override `toString` since we are not separating any value.